sktime/utils/validation/__init__.py

```diff
@@ -38,12 +38,45 @@ def check_n_jobs(n_jobs):
         return n_jobs
 
 
-def check_window_length(window_length, name="window_length"):
+def check_window_length(window_length, y=None, name="window_length"):
```

Comment:
rather than passing `y`, I would pass `n_timepoints` and compute `n_timepoints = y.shape[0]` outside of the function.

Reply:
I have edited the function and computed `n_timepoints` outside the function. Should I go ahead with the test?

Reply:
Yes please write some unit testing for this! 👍

Reply:
Hello Markus. I have started writing the test
but I am unsure if I am also going to include asserts when the window_length is not a valid input.

Reply:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/13348874/116110318-7eb0c880-a6ad-11eb-9244-f8f70f9fd286.png">

Reply:
For invalid inputs you can use something like this: 
```python
with pytest.raises(ValueError): 
    check_window_length("bad_window_length")
```

Reply:
You can also use `pytest.mark.parametrize()` to iterate over a test function

Reply:
`from sktime.utils.validation.__init__` is the same as `from sktime.utils.validation`